lms/events/event.py

```diff
@@ -101,3 +103,60 @@ def _get_assignment_id(self):
             return assignment.id
 
         return None
+
+
+@dataclass
+class AuditTrailEvent(BaseEvent):
+    type: EventType.Type = EventType.Type.AUDIT_TRAIL
+
+    instance: BASE = None
+    """Object for which we are tracking changes"""
```

Comment:
I don't think the instance is safe in general in events as we can't handle these out of band of the transaction without weird things happening. Keeping `model` and `id` would probably be safer?